controller/Makefile

```diff
@@ -4,6 +4,8 @@ include ../includes.mk
 
 all: build run
 
+SHELL_SCRIPTS = build.sh $(wildcard bin/*) $(wildcard migrations/data/*.sh)
```

Comment:
Something easier and more dynamic is to use `$(shell find $PWD -name '*.sh')`. That way we won't have to update this if we introduce new directories. Lazy searching ftw